Packs/CommonPlaybooks/ReleaseNotes/2_4_12.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Dedup - Generic v4
+
+- Added the *'DBotFindSimilarIncidentsByIndicators'* context key to the playbook outputs.
+- Updated tasks and playbook descriptions.
+- Updated the script arguments of the playbook tasks from simple to complex values.
```

Comment:
you can remove this line as its a yml change and not effecting the playbook itself.

Reply:
removed